tests/test_helpers.py

```diff
@@ -431,14 +440,38 @@ def test_proxies_from_env_http_proxy_for_https_proto(mocker) -> None:
     assert ret['https'].proxy == url
     assert ret['https'].proxy_auth is None
 
+def test_proxies_from_env_http_proxy_for_wss_proto(mocker) -> None:
+    url = URL('http://aiohttp.io/path')
+    mocker.patch.dict(os.environ, {'wss_proxy': str(url)})
```

Comment:
It's preferrable to use a built-in monkeypatch fixture:
```suggestion
def test_proxies_from_env_http_proxy_for_wss_proto(monkeypatch) -> None:
    url = URL('http://aiohttp.io/path')
    monkeypatch.setenv('wss_proxy', str(url))
```